serenity-develop/features/blockstack/step_definitions/explorer.steps.ts

```diff
@@ -0,0 +1,122 @@
+import {browser, by, protractor} from 'protractor';
+import {HomePage} from './spec/HomePage';
+const chai = require('chai');
+chai.use(require('chai-as-promised'));
+
+const expect = chai.expect;
+
+module.exports = function myStepDefinitions () {
+    const homePage: HomePage = new HomePage();
+    this.setDefaultTimeout(60 * 1000);
+
+    this.Given(/^user is on home page$/, async ()=> {
+        homePage.openHomePage();
+      });
+      this.Then('user enter the name in search bar', async ()=> {
+        await homePage.enterNameInSearchBar("aaronbailey.id");
```

Comment:
`aaronbailey.id` is an id that may expire in the future (if the owner doesn't renew the name) -- using a subdomain like name (e.g., `aaron.id.blockstack`) or a name like `jude.blockstack` which doesn't expire will prevent this from raising a false error in the future.

Reply:
Updated